governance/votes/Executive Vote - February 4, 2022.md

```diff
@@ -0,0 +1,104 @@
+- **12,000 DAI** transferred to [Feedblack Loops LLC](https://vote.makerdao.com/address/0x845b36e1e4f41a361dd711bda8ea239bf191fe95) at [0x80882f2A36d49fC46C3c654F7f9cB9a2Bf0423e1](https://etherscan.io/address/0x80882f2A36d49fC46C3c654F7f9cB9a2Bf0423e1).
+- **8,620 DAI** transferred to [MakerMan](https://vote.makerdao.com/address/0x22d5294a23d49294bf11d9db8beda36e104ad9b3) at [0x9AC6A6B24bCd789Fa59A175c0514f33255e1e6D0](https://etherscan.io/address/0x9AC6A6B24bCd789Fa59A175c0514f33255e1e6D0).
+- **4,807 DAI** transferred to [monetsupply](https://vote.makerdao.com/address/0x45127ec92b58c3a89e89f63553073adcaf2f1f5f) at [0x4Bd73eeE3d0568Bb7C52DFCad7AD5d47Fff5E2CF](https://etherscan.io/address/0x4Bd73eeE3d0568Bb7C52DFCad7AD5d47Fff5E2CF)
+- **3,795 DAI** transferred to [ACREinvest](https://vote.makerdao.com/address/0x4d3ac33ab1dd7b0f352b8e590fe8b62c4c39ead5) at [0x5b9C98e8A3D9Db6cd4B4B4C1F92D0A551D06F00D](https://etherscan.io/address/0x5b9C98e8A3D9Db6cd4B4B4C1F92D0A551D06F00D).
+- **889 DAI** transferred to [JustinCase](https://vote.makerdao.com/address/0xcdb792c14391f7115ba77a7cd27f724fc9ea2091) at [0xE070c2dCfcf6C6409202A8a210f71D51dbAe9473](https://etherscan.io/address/0xE070c2dCfcf6C6409202A8a210f71D51dbAe9473).
+- **641 DAI** transferred to [GFX Labs](https://vote.makerdao.com/address/0xf60d7a62c98f65480725255e831de531efe3fe14) at [0xa6e8772af29b29B9202a073f8E36f447689BEef6](https://etherscan.io/address/0xa6e8772af29b29B9202a073f8E36f447689BEef6).
+
+Details on the breakdown of delegate compensation for the month of January can be found on the MakerDAO forum [here](https://forum.makerdao.com/t/recognized-delegate-compensation-breakdown-january-2022/13001).
+
+### Emergency Shutdown Module Threshold Increase
+
+As per this [governance poll](https://vote.makerdao.com/polling/QmQSVmrh), the following change to the [Emergency Shutdown Module](https://makerdao.world/en/learn/governance/emergency-shutdown) (ESM) will be made if this executive proposal passes.
+
+- Raise the threshold to trigger the [Emergency Shutdown Module](https://makerdao.world/en/learn/governance/emergency-shutdown) by 25,000 MKR from **75,000 MKR** to **100,000 MKR**.
+
+For further information, please see the initial Signal Request on the MakerDAO forum [here](https://forum.makerdao.com/t/signal-request-raise-threshold-for-emergency-shutdown-module/12208).
+
+Further details about the changes being made to the Emergency Shutdown Module have been posted by the Protocol Engineering Core Unit on the MakerDAO forum [here](https://forum.makerdao.com/t/moving-the-esm-threshold-increase-to-next-week/12895).
+
+### MOMC Parameter Changes
+
+As per this [governance poll](https://vote.makerdao.com/polling/QmWReBMh) on the latest MakerDAO Open Market Committee [proposal](https://forum.makerdao.com/t/parameter-changes-proposal-ppg-omc-001-2022-01-27/12883), the following parameter changes will be made in the Maker Protocol if this executive proposal passes:
+
+**Stability Fee Decreases**
```

Comment:
We're not leaving an empty line here but we are elsewhere (e.g., "**Maximum Debt Ceiling Changes**\n\n").

Reply:
Also not sure if we should use headers instead of just bold?

Reply:
Fixed empty line, the bold is just for consistency with the last exec that had MOMC changes, not particularly wedded to it, but best to be consistent IMO
For reference: https://github.com/makerdao/community/blob/master/governance/votes/Executive%20Vote%20-%20January%2021,%202022.md

Reply:
In reality, the difference between #### and bold is pretty small anyway